addons/survey/models/survey_survey.py

```diff
@@ -639,7 +639,7 @@ def get_input_summary(self, question, current_filters=None):
             ('question_id', '=', question.id)
         ]
         if current_filters:
-            domain = expression.AND([[('id', 'in', current_filters)], domain])
+            domain = expression.AND([[('user_input_id.id', 'in', current_filters)], domain])
```

Comment:
@djs-metrum i don't think that the .id is needed

Reply:
`current_filters` contains the ids of user_inputs, will it work if we just put user_input_id without .id ?

Reply:
yes, in a domain we always use the record ids and we don't need to prefix the field name by .id